keyboards/converter/ps2_usb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,30 @@
+#include "ps2_usb.h"
```

Comment:
This needs to be changed 
```suggestion
#include QMK_KEYBOARD_H
```